diff --git a/node_modules/electron-updater/out/BaseUpdater.js b/node_modules/electron-updater/out/BaseUpdater.js
index f441eaa..b4e8f0a 100644
--- a/node_modules/electron-updater/out/BaseUpdater.js
+++ b/node_modules/electron-updater/out/BaseUpdater.js
@@ -14,11 +14,11 @@ class BaseUpdater extends AppUpdater_1.AppUpdater {
         // If NOT in silent mode use `autoRunAppAfterInstall` to determine whether to force run the app
         const isInstalled = this.install(isSilent, isSilent ? isForceRunAfter : this.autoRunAppAfterInstall);
         if (isInstalled) {
-            setImmediate(() => {
+            setTimeout(() => {
                 // this event is normally emitted when calling quitAndInstall, this emulates that
                 require("electron").autoUpdater.emit("before-quit-for-update");
                 this.app.quit();
-            });
+            }, 1000);
         }
         else {
             this.quitAndInstallCalled = false;
